website/app/styles/spacing/index.scss

```diff
@@ -78,31 +78,45 @@ pre[class*="language-"].doc-markdown-pre.doc-code-block__code-snippet {
   margin: 16px 0 32px;
 }
 
-.doc-code-block + .doc-code-block,
-.doc-code-block + pre[class*="language-"].doc-markdown-pre.doc-code-block__code-snippet,
-pre[class*="language-"].doc-markdown-pre.doc-code-block__code-snippet + .doc-code-block,
-pre[class*="language-"].doc-markdown-pre.doc-code-block__code-snippet + pre[class*="language-"].doc-markdown-pre.doc-code-block__code-snippet {
+.doc-code-block+.doc-code-block,
+.doc-code-block+pre[class*="language-"].doc-markdown-pre.doc-code-block__code-snippet,
+pre[class*="language-"].doc-markdown-pre.doc-code-block__code-snippet+.doc-code-block,
+pre[class*="language-"].doc-markdown-pre.doc-code-block__code-snippet+pre[class*="language-"].doc-markdown-pre.doc-code-block__code-snippet {
   // we want to reduce the spacing between to consecutive code blocks
   margin-top: -16px;
 }
 
+// TABLES
+
+.doc-markdown-table {
+  margin-top: 32px;
+}
+
 
 // SPECIAL CASES
 
 // Banner + Heading (any) => Banner must have margin-bottom = 32px
 
 .doc-banner,
 .doc-do-dont {
-  & + .doc-markdown-h2,
-  & + .doc-markdown-h3,
-  & + .doc-markdown-h4,
-  & + .doc-markdown-h5,
-  & + .doc-markdown-h6,
-  & + .doc-text-h2,
-  & + .doc-text-h3,
-  & + .doc-text-h4,
-  & + .doc-text-h6,
-  & + .doc-text-h5 {
+
+  &+.doc-markdown-h2,
+  &+.doc-markdown-h3,
+  &+.doc-markdown-h4,
+  &+.doc-markdown-h5,
+  &+.doc-markdown-h6,
+  &+.doc-text-h2,
+  &+.doc-text-h3,
+  &+.doc-text-h4,
+  &+.doc-text-h6,
+  &+.doc-text-h5 {
     margin-top: 32px;
   }
 }
+
+// Hack for adding spacing to HDS components within a paragraph tag
+// PLEASE REMOVE THIS, IT'S TERRIBLE
+
+.doc-markdown-p>*[class*="hds-"] {
```

Comment:
This is not possible, it's too generic. It will be fixed more holistically in this ticket: https://hashicorp.atlassian.net/browse/HDS-1366

Reply:
Yep, I figured. Hack for illustrative purposes and so I could "see" what it looked like in the browser.